### DISCUSSION TITLE
How to retrieve healthy slave nodes from SentinelPool and load balance GET requests

### DISCUSSION THREAD

**Discussion by RaghuGuruvayurappan** [xetorthio/jedis#2921] [General]
Hi Team,

Currently, we are retrieving the master node from SentinelPool and using it to write data. Since read operations are very high, we want to distribute read to all slave nodes.
Is it feasible with Jedis?

Thanks

**Comment by sazzad16** [comment]
@RaghuTrivandrum Sorry, we don't have this feature implemented at this moment.

**Reply by luger1990** [reply]
@sazzad16 does jedis team have a plan support this?

**Reply by sazzad16** [reply]
@luger1990 We don't have enough resource to work on this. But we'll welcome a PR.
